lldb/source/Plugins/Process/elf-core/ThreadElfCore.cpp

```diff
@@ -541,9 +542,18 @@ size_t ELFLinuxSigInfo::GetSize(const lldb_private::ArchSpec &arch) {
   }
 }
 
+static bool IsSignalWithAddrValue(int signo) {
+  // SIGILL, SIGFPE, SIGSEGV, SIGBUS, SIGTRAP
+  // We can't use the enum here because it may not be available on windows or
+  // other platforms. We should make an LLDB platform agnostic enum for this
```

Comment:
We already have that. This data is encoded in the `process->GetUnixSignals()` object. You just need to plumb it to the right place.